app/views/users/_form_new.html.erb

```diff
@@ -0,0 +1,39 @@
+
+<%= simple_form_for @user, html: { class: 'form-horizontal', multipart: true } do |f| %>
+       
+    <%= f.input :username, input_html: {class: "form-control"}, wrapper_html: {class: 'form-group'} %>
+    
+    <%= f.input :email, input_html: {class: 'form-control'}, wrapper_html: {class: 'form-group'}%>
+
+
+    <%= f.input :first_name, input_html: {class: "form-control"}, wrapper_html: {class: 'form-group'} %>
+
+    <%= f.input :last_name, input_html: {class: "form-control"}, wrapper_html: {class: 'form-group'} %>
+
+
+    <%= f.input :password, input_html: {class: "form-control"}, wrapper_html: {class: 'form-group'} %>
+
+    <%= f.input :password_confirmation, input_html: {class:'form-control'}, wrapper_html: {class: 'form-group'}%>
+
+
+    <%= f.input :phone, input_html: {class: "form-control"}, wrapper_html: {class: 'form-group'} %>
+
+    <%= f.input :job_title, input_html: {class: "form-control"}, wrapper_html: {class: 'form-group'} %>
+    
+    <%= f.input :organization_id, collection: @organizations, input_html: {class: "form-control"}, wrapper_html: {class: 'form-group'} %>
+    <div>&nbsp</div>
+
+    <% if logged_in? && current_user.role?(:admin) %>
+        <%= f.input :role, collection: User::ROLES, wrapper_html: {class: 'form-group'}, prompt: true %>
```

Comment:
I created a different form for the users/new and user/:id/edit.

So for the edit action, users are not allowed to change their emails and usernames directly. They have to contact admins for help